client/lib/hoverIntent/README.md

```diff
@@ -0,0 +1,64 @@
+## Hoverintent
+
+"hoverIntent is a plug-in that attempts to determine the user's intent"
+
+## Props
+
+```onMouseOver``` : The callback to fire on mouse over event
```

Comment:
Should we change those to resemble more the format we use for documenting props?